src/base_routines.f90

```diff
@@ -883,7 +883,7 @@ END SUBROUTINE BASE_ROUTINES_INITIALISE
   !================================================================================================================================
   !
 
-  !>Sets diagnositics off. \see BASE_ROUTINES::DIAGNOSTICS_SET_ON,OPENCMISS::CMISSDiagnosticsSetOn
+  !>Sets diagnositics off. \see BASE_ROUTINES::DIAGNOSTICS_SET_OFF,OPENCMISS::CMISSDiagnosticsSetOff
   SUBROUTINE DIAGNOSTICS_SET_OFF(ERR,ERROR,*)
```

Comment:
@vraj004 Just for future reference. The OPENCMISS::CMISSxxx parts are old and obviously haven't been changed yet since the rename. The correct \see should be \see OpenCMISS::Iron::cmfe_DiagnosticsSetOn etc.